app/api/sessions.py

```diff
@@ -31,6 +31,7 @@
     'id': fields.Integer(required=True),
     'name': fields.String(),
     'color': fields.Color(),
+    'font_color': fields.Color(),
```

Comment:
Don't allow user to set font color. Font color needs to be a readonly attribute on the API